project/AppDependencies.scala

```diff
@@ -6,14 +6,17 @@ import sbt._
 object AppDependencies {
 
   val compile = Seq(
-    "uk.gov.hmrc"             %% "bootstrap-backend-play-27"  % "3.4.0"
+    "uk.gov.hmrc" %% "bootstrap-backend-play-27" % "3.4.0",
+    "org.apache.axis2" % "axis2-kernel" % "1.7.9",
+    "wsdl4j" % "wsdl4j" % "1.6.1"
```

Comment:
Is this dependency necessary ->  I hadn't pulled it in on import-control-api-stub, only the axis2-kernel dependency; what does it do?

Reply:
Ahh I see you're using it to read the wsdl to check the wsdl operation